fuzz/src/lib.rs

```diff
@@ -34,7 +37,8 @@ pub fn generate_valid_operation(input: &[u8], schema_path: &'static str) -> Resu
         &mut u,
         Document::try_from(tree.document()).expect("tree should not have errors"),
     )?;
-    let operation_def = gql_doc.operation_definition()?.unwrap();
+    let operation_def: String = gql_doc.operation_definition()?.unwrap().into();
+    let doc: String = gql_doc.finish().into();
 
-    Ok(operation_def.into())
+    Ok((operation_def, doc))
```

Comment:
is doc used somewhere?

Reply:
Yep it's used in the apollo_router_studio_interop.rs fuzz target.

Reply:
Oh ok, i thought it was removed, lgtm!